gdc_client/upload/client.py

```diff
@@ -216,20 +216,24 @@ def get_files(self, action='download'):
                     self.file_entities.append(file_entity)
                     continue
 
-                path = f.get('path') or '.'
-                filename = f.get('file_name') or self.metadata['file_name']
-                file_entity.file_path = os.path.join(path, filename)
+                # first check local_file_path
+                # second check file_name
+                # https://github.com/NCI-GDC/gdcapi/pull/426#issue-146068652
+                file_entity.file_path = f.get('local_file_path') or \
```

Comment:
Please validate that `local_file_path` actually contains the file name and not only a path to the location of the file.

According to the variable name I am under the impression that you would have to append the file name to the path like this:
```
path = f.get('local_file_path') or '.'
filename = f.get('file_name') or self.metadata['file_name']
file_entity.file_path = os.path.join(path, filename)
```